src/tapis-redux/src/streams/sites/reducer.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  SitesReducerState,
+  SitesListingRequestPayload,
+  SitesListingSuccessPayload,
+  SitesListingFailurePayload
+} from './types';
+import {
+  updateList,
+  setRequesting,
+  setFailure,
+  getEmptyListResults,
+  TapisListResults
+} from 'tapis-redux/types/results'
+import { TAPIS_DEFAULT_SITES_LISTING_LIMIT } from 'tapis-redux/constants/tapis';
+import * as ACTIONS from './actionTypes';
+import * as Streams from "@tapis/tapis-typescript-streams";
+
+
+const emptyResults = getEmptyListResults(TAPIS_DEFAULT_SITES_LISTING_LIMIT);
+
+export const initialState: SitesReducerState = {
+  sites: { ...emptyResults }
```

Comment:
Is there a hierarchical way of storing these objects in the reducer? Does it make sense to have { [site: string]: { projects, instruments } or a similar concept? Similar to [File listings belonging to systems](https://github.com/tapis-project/tapis-ui/blob/bf67b16b177b2c7d58b3d848fbf5bf581107a8a5/src/tapis-redux/src/files/types.ts#L11)